protocol/consistencychecks_test.go

```diff
@@ -3,29 +3,28 @@ package protocol
 import "testing"
 
 var (
-	key = []byte("key")
+	alice = "alice"
+	bob   = "bob"
+	key   = []byte("key")
 )
 
-func doRequestAndVerify(d *ConiksDirectory, cc *ConsistencyChecks,
-	requestType int, name string) error {
-	switch requestType {
-	case RegistrationType:
-		request := &RegistrationRequest{
-			Username: name,
-			Key:      key,
-		}
-		res, _ := d.Register(request)
-		return cc.HandleResponse(requestType, res, name, key)
-	case KeyLookupType:
-		request := &KeyLookupRequest{
-			Username: name,
-		}
-		res, _ := d.KeyLookup(request)
-		return cc.HandleResponse(requestType, res, name, key)
-	case MonitoringType:
-	case KeyLookupInEpochType:
+func registerAndVerify(d *ConiksDirectory, cc *ConsistencyChecks,
```

Comment:
You're saying that since it's covered there, it's redundant here.  That's true.

As a counterpoint though, the code currently looks like,

```
// test error name existed
if registerAndVerify(d, cc, alice, key) != CheckPassed {
  t.Fatal("Unexpected verification result")
}
```

As someone reading it for correctness, it's really not obvious that `CheckPassed` means that `ReqNameExisted`.  To me, `CheckPassed` just means that the response was consistent.  It doesn't tell me what it was though.